pypeit/scienceimage.py

```diff
@@ -147,7 +173,9 @@ def __init__(self, spectrograph, file_list, det = None, objtype='science', scidx
         self.skymask = None
         self.objmask = None
         self.outmask = None
+        # TODO: (KBW) I'd like to change the bitmask nomenclature
```

Comment:
Please do.

Reply:
Done.  @jhennawi please make sure you agree with this change.